ntfs_fr.md

```diff
@@ -528,6 +529,21 @@ Colonne | Type | Contrainte | Commentaire
 address_id | chaine | Requis | Identifiant de l'adresse
 street_name | chaine | Requis | Nom de la voierie
 house_number | chaine | Optionnel | Numéro du seuil
+admin_level_8_id | chaine | Optionnel | Identifiant de l'administration de niveau 8 dans le fichier `administrative_regions.txt`
+admin_level_9_id | chaine | Optionnel | Identifiant de l'administration de niveau 9 dans le fichier `administrative_regions.txt`
+admin_level_10_id | chaine | Optionnel | Identifiant de l'administration de niveau 10 dans le fichier `administrative_regions.txt`
+
+### administration_regions.txt (optionnel)
```

Comment:
Not sure, but how about `administrative_regions.txt` instead of `administration_regions.txt`?

See [here](https://github.com/osm-without-borders/cosmogony/blob/2619a1602483744b14365fb3c6eeb27ba68b7f18/README.md?plain=1#L72) and [here](https://github.com/hove-io/mimirsbrunn/blob/427d5d6dd0641259d77209e25a3a877abc5bc1db/docs/assets/tbl/fields-admin.md?plain=1#L4) for why I suggest that modification.

Reply:
we have already considered this solution, but there won't be any levels other than 8, 9 and 10.
We've decided to keep things simple for the moment.
but the specifications can be changed later i the needs change